testgrid/specs/storage-migration.yaml

```diff
@@ -336,8 +323,6 @@
       version: 1.6.x
     flannel:
       version: 0.21.x
-    registry:
-      version: 2.8.1
```

Comment:
this test case absolutely should work and does not require S3, _even with registry_

I do not support removing registry from this test

If the preflight is catching this, the preflight is too restrictive

Reply:
@laverya we discussed that. I proposed change the check and you and @emosbaugh do not agree with it.
See: https://replicated.slack.com/archives/C0499EGNPNY/p1679944354578159

Reply:
The appropriate check here isn't "is kots running with s3override"

it's "is there currently an object store that we would be migrating Registry away from"